WebHostLib/tracker.py

```diff
@@ -1207,6 +1219,9 @@ def __renderGenericTracker(multisave: Dict[str, Any], room: Room, locations: Dic
     for order_index, networkItem in enumerate(ordered_items, start=1):
         player_received_items[networkItem.item] = order_index
 
+    app.jinja_env.filters["location_name"] = lambda loc: collections.ChainMap(lookup_any_location_id_to_name, custom_locations).get(loc, loc)
+    app.jinja_env.filters["item_name"] = lambda item: collections.ChainMap(lookup_any_item_id_to_name, custom_items).get(item, item)
```

Comment:
this still sets the global filter right before rendering. Jinja can be called in a multithreaded fashion.